### PR TITLE
Fix Leaderboard Sorting Issue

### LINKED ISSUE #1756: Gulp Staging Fails
The new change introduced in #1755 breaks the gulp staging task. Please see the error log below:

```
deshraj at DESHRAJs-MacBook-Pro in ~/Documents/EvalAI on staging [$]
$ docker-compose run nodejs gulp staging
[05:23:44] Using gulpfile /code/gulpfile.js
[05:23:44] Starting 'staging'...
[05:23:44] Starting 'clean'...
[05:23:44] Finished 'clean' after 174 ms
[05:23:44] Starting '<anonymous>'...
[05:23:44] Finished '<anonymous>' after 917 μs
[05:23:44] Starting 'vendorcss'...
[05:23:44] Starting 'vendorjs'...
[05:23:44] Starting 'css'...
[05:23:44] Starting 'js'...
[05:23:44] Starting 'html'...
[05:23:44] Starting 'images'...
[05:23:44] Starting 'fonts'...
[05:23:45] Finished 'css' after 1.23 s
[05:23:45] Finished 'vendorcss' after 1.25 s
[05:23:46] 'js' errored after 1.39 s
[05:23:46] GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected token operator «=», expected punc «,»
File: /code/frontend/src/js/controllers/controllers.min.js
Line: 1007
[05:23:46] 'staging' errored after 1.57 s
[05:23:46] The following tasks did not complete: vendorjs, html, images, fonts
[05:23:46] Did you forget to signal async completion?
```

cc: @live-wire @RishabhJain2018

### PR DESCRIPTION
**Pull request by live-wire**
Learnings: 
- While passing arguments to functions, it is not the best way to give default values to arguments (Python like syntax / Typescript like syntax) at-least for legacy AngularJS.

Please review this @deshraj @RishabhJain2018

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@live-wire I tested it and it seems to break again. Please see the screenshot below.
Also, Please fix it asap as we need this to be fixed urgently.
screen shot 2018-08-05 at 9 09 38 pm

**Comment by live-wire** [comment]
_dev:runserver_ uses `configDev` which should instead use `configStaging`. 
So I created a new task _staging:runserver_ which uses the staging config.
(and similarly for _prod_)

File: gulpfile.js [hunk 1]
```diff
@@ -324,4 +324,12 @@ gulp.task('dev:runserver', gulp.series(function(done) {
     done();
```
Comment (RishabhJain2018) [inline gulpfile.js]: @live-wire Please remove this file from here and update the PR description. I have opened a PR #1771 for the gulp staging fix.

**Comment by live-wire** [comment]
@RishabhJain2018 Removed the new gulp tasks from this PR 👍

**Comment by deshraj** [comment]
@live-wire can you please update this PR ? We should get this merged soon.

**Comment by RishabhJain2018** [comment]
@live-wire You have to rebase your branch and push again.
Documentation for git rebase: blog.algolia.com/master-git-rebase
Please feel free to ask if you have some doubts.

**Comment by live-wire** [comment]
@RishabhJain2018 @deshraj Done! 🚗

**Pull request merged**